packages/gatsby-core-utils/src/get-config-store.ts

```diff
@@ -15,3 +15,5 @@ module.exports = () => {
 
   return config
 }
+
+export default getConfigStore
```

Comment:
please change this to a named export! ❤️

Reply:
Done. Going to update #22053 accordingly